### DISCUSSION TITLE
Why not running my bash_profile on MacOS ?

### DISCUSSION THREAD

**Discussion by kaihendry** [zellij-org/zellij#1733] [Q&A]
My bash_profile isn't sourced on my MacOS machine... any ideas why? https://s.natalian.org/2022-09-15/zellij.png

I don't quite understand how the shell is configured.

**Comment by lazzurs** [comment]
The answer for anyone else hitting this from a search engine is that only your path settings should be in .bash_profile and everything else should be in .bashrc which will get run the way zellij starts the shell.
